addons/survey/models/survey_question.py

```diff
@@ -130,6 +130,23 @@ def default_get(self, fields):
         'survey.user_input.line', 'question_id', string='Answers',
         domain=[('skipped', '=', False)], groups='survey.group_survey_user')
 
+    # Conditional display
+    is_conditional = fields.Boolean(string='Conditional Display',
+                                    help="""If checked, this question will be displayed only if the specified
+                                            conditional answer have been selected in a previous question""")
+    conditional_question_id = fields.Many2one('survey.question', string="Conditional Question",
```

Comment:
conditional_parent_id ?

Reply:
or triggering_question_id and triggering_answer_id ?